src/__tests__/events.js

```diff
@@ -33,7 +33,7 @@ const eventTypes = [
   },
   {
     type: 'Focus',
```

Comment:
While we're at it would you be so kind and change it to `Event`. These are not focus events.

Reply:
Maybe I should change 

```js
  {
    type: 'Focus',
    events: ['focus', 'blur', 'focusIn', 'focusOut'],
    elementType: 'input',
  },
  {
    type: 'Form',
    events: ['focus', 'blur'],
    elementType: 'input',
  },
  {
    type: 'Focus',
    events: ['change', 'input', 'invalid'],
    elementType: 'input',
  },
  {
    type: 'Focus',
    events: ['submit', 'reset'],
    elementType: 'form',
  },
```

to

```js
  {
    type: 'Focus',
    events: ['focus', 'blur', 'focusIn', 'focusOut'],
    elementType: 'input',
  },
  {
    type: 'Input',
    events: ['change', 'input', 'invalid'],
    elementType: 'input',
  },
  {
    type: 'Form',
    events: ['submit', 'reset'],
    elementType: 'form',
  },
```

so that there is no repetition and no `Event Events` description?

Reply:
Yes, I think that would be appropriate. Thank you!